dist/metadata.json

```diff
@@ -2,8 +2,8 @@
   "uuid": "gTile@vibou",
   "name": "gTile",
   "description": "Tile windows on a grid",
-  "shell-version": ["45", "46"],
```

Comment:
Why are previous shell versions no longer supported?

Reply:
Because there was, technically, a [breaking change](https://gitlab.gnome.org/GNOME/gnome-shell/-/commit/0f30bfdd531ed0e8ed78bda74d51b7f22872a31b) in the gnome-shell API. 

Also see https://gjs.guide/extensions/upgrading/gnome-shell-47.html
Also see https://github.com/gjsify/gnome-shell/pull/49

I opted to handle this as a breaking change (although it wouldn't be necessary) because we hadn't had any changes in the repository anyways.